indico/modules/rb/templates/booking_list.html

```diff
@@ -128,14 +130,20 @@ <h3>Room bookings</h3>
                                         </span>
                                     {% endif %}
                                 </td>
-                                <td class="action-column">
-                                    {{ booking_details_link(reservation) }}
-                                </td>
+                                {% if is_rb_visible %}
+                                    <td class="action-column">
+                                        {{ booking_details_link(reservation) }}
+                                    </td>
+                                {% endif %}
                             </tr>
                         {% endfor %}
                     </tbody>
                 </table>
             </div>
+        {% else %}
+            <div class="text">
+                <h3>{%- trans %}There are not yet bookings for this event.{% endtrans -%}</h3>
```

Comment:
```suggestion
                <h3>{%- trans %}There are no room bookings for this event yet.{% endtrans -%}</h3>
```

("there are not yet" sounds very weird)